buildlib/tools/builds.sh

```diff
@@ -377,6 +377,22 @@ build_fuse() {
 	fi
 }
 
+#
+# Build with static library
+#
+build_static() {
+	${WORKSPACE}/contrib/configure-devel --prefix=$ucx_inst
+	$MAKEP
+	$MAKEP install
+
+	PKG_CONFIG_PATH=$ucx_inst/lib/pkgconfig $MAKE -C test/apps/uct_info LDFLAGS="-Wl,-rpath=$ucx_inst/lib"
+
+	./test/apps/uct_info/uct_info
+	./test/apps/uct_info/uct_info_static
+
+	$MAKE -C test/apps/uct_info test
```

Comment:
why not just run check_tls.sh directly?

Reply:
> why not just run check_tls.sh directly?

because check_tls.sh scripts requires binaries in current work directory (there is no good way to know where binaries are located in case if VPATH is used)

Reply:
Can we pass it as command line parameter passed from azure scripts?

Reply:
removed ```test``` target, script is launched directly